libs/types/test/election.ts

```diff
@@ -127,7 +127,7 @@ export const primaryElection: Election = {
     ...election.ballotStyles.map((bs) => ({
       ...bs,
       id: `${bs.id}R`,
-      groupId: `${bs.id}D`,
+      groupId: `${bs.id}R`,
```

Comment:
unrelated bug in a test fixture that was being hidden by the normalizeVxf function changing ballot style ids